### DISCUSSION TITLE
is the feature of this plugin possible without the plugin?

### DISCUSSION THREAD

**Discussion by Julian-1-2-3-4-5** [ehudhala/logseq-plugin-daily-todo#6] [General]
because this plugin didn't work as i wanted it to, i used the default queries for the journal in the config.edn
with the query:
```
{:title "📅 NEXT"
    :query [:find (pull ?h [*])
            :in $ ?start ?limit
            :where
            [?h :block/marker ?marker]
            [(contains? #{"NOW" "LATER" "TODO"} ?marker)]
            ; I want to see TODOs from all pages, not just journals
            ; and I ignore start and limit, just show all.
            ; [?h :block/page ?p]
            ; [?p :block/journal? true]
            ; [?p :block/journal-day ?d]
            ; [(>= ?d ?start)]
            ; [(< ?d ?limit)]
]
    :inputs [:100d :7d-after]
    :collapsed? false}
```
The benefit of this is:
- that we don't have the Problem that this plugin sometimes has, that it moves more than just the tasks. 
- that it keeps our tasks on the day they where created and just always shows them on todays journal page which is more in line with the logseq workflow

**Comment by Julian-1-2-3-4-5** [comment]
I don't know much about plugins or if this could be what the plugin could do instead, but it might be a good idea to include this in the description, because i think it is mor in line with what people want this plugin to do and doesn't even need the plugin

**Comment by icemaxx5** [comment]
how do i add this to the journal default query? sorry im a total noob!
